CONTRIBUTORS.md

```diff
@@ -6,21 +6,34 @@ This is a very young project, and the codebase is moving quickly -- but there ar
 
 Please use the Github issue tracker for all issues with, and questions about, the project. 
 
-If you have any questions, please file an issue and we'll answer it as soon as we can. Make your question or issue
-clear in the title, and be sure to look through the issues just in case someone else has asked the same questions.
+If you have any questions, submit them to the [Ansible Container Mailing List](https://groups.google.com/forum/?hl=en-GB#!forum/ansible-container) or file an issue,
+and we'll answer it as soon as we can. When filing an issue make your question is clear in the title, and be sure to search existing issues to see if someone else 
+has asked the same question.
 
 ## Pull requests
 
-Pull requests are always welcome. We may not necessarily accept them, but we will happy to discuss them with you. Please
-observe the following when submitting a PR:
+Code submissions are accepted via pull request. Pull requests are always welcome! We may not accept every pull request, but we are always happy to review and 
```

Comment:
Style: three sentences, four uses of the term `pull request`. Pronouns are awesome.